worldedit-core/src/main/java/com/sk89q/worldedit/blocks/BlockType.java

```diff
@@ -625,62 +641,64 @@ public boolean canPassThrough() {
     /**
      * HashSet for centralTopLimit.
      */
-    private static final Map<Integer, Double> centralTopLimit = new HashMap<Integer, Double>();
+    private static final Map<Integer, Double> centralTopLimit;
```

Comment:
@TomyLobo This `Map` has conflicts, should it be having those or does this code need to be rewritten?